releases/23.05.0/23050 feature list.md

```diff
@@ -0,0 +1,196 @@
+* The texture streaming support has been expanded to Vulkan backend. Now we have image streaming control which can load or evict texture mips automatically at runtime based on the streaming image pool's budget.
+* The shader library, material system, and builders have been updated to support a layer of abstraction between lighting and materials, called the material pipeline. It uses new builders and new features from the asset system to build unique shaders for each render pipeline. Material types can provide custom data structures and evaluator functions for vertex, pixel, geometry, and lighting stages. The builder inserts any of the custom include files with overridden data types and functions as it builds all of the combinations of shaders listed in the material pipeline.  
+	* The result of this is a concrete material type referencing all of the different combinations of shaders for each pipeline, with the customization stitched in. All of this is saved to the intermediate assets folder and processed by the AP to generate Final product material types and shaders. The material pipeline also includes a Lua script that will be invoked whatever material properties or the active render pipeline changes to enable or disable shaders based on the current state.  
+	* All of the PBR as well as other shaders and material types included with atom have been converted to the system.  
+	* All of the ASV samples and tests have also been updated to use this system. ASV also includes different experimental render and material pipelines to demonstrate how existing materials that use this system can be applied to a standard, deferred rendering pipeline.
+	* Known issues:  
+		* It takes more time to compile shaders for all of the active pipelines.
+		* Material editor and material canvas crash switching to Multiview at XR pipelines.
+		* The organization of all of the shader and material files needs to be simplified to make it easier to find shader and material files.
+		* There are several to do items as a result of these changes that need to be addressed.
+
+
+## sig-network
+
+* Adds new warnings when client and server have differences in networked code and properties to aid debugging of these issues.
+* Provides method for code separation between client and server side code to prevent any exposure of server side logic to clients. Provides a new Unified Launcher target to aid local testing.
+* Provides a more out-of-the-box demonstration of Amazon GameLift using the new multiplayer sample with workflow refinements to make it easier to setup and use.
+
+ 
+## sig-operations
```

Comment:
@aFinchy Correct me if I'm wrong, but I think we don't need SIG operations in the features list because the SIG is more an operation of everything than the engine.

Reply:
I will leave it here. It can always be removed later.